spec.html

```diff
@@ -1186,23 +1185,23 @@ <h1>ToElementDescriptors ( _elementsObject_ )</h1>
     </emu-clause>
 
     <emu-clause id=sec-to-element-descriptor aoid=ToElementDescriptor>
-      <h1>ToElementDescriptor ( _descriptor_ )</h1>
-      <p>With parameter _descriptor_, returns a Record containing three values: { [[Element]]: a class Element Descriptor, [[Extras]]: an iterable of other elements, [[Finisher]]: a Function or *undefined* }.</p>
+      <h1>ToElementDescriptor ( _elementObject_ )</h1>
```

Comment:
Nit: Could you call this elementDescriptor, and the other descriptor propertyDescriptor? We consistently use the word "descriptor" to refer to element descriptors as well.